docker-compose.yml

```diff
@@ -4,17 +4,17 @@ services:
   postgres_development:
     image: postgres:13.1
     environment:
-      - POSTGRES_USER=user
-      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=
       - POSTGRES_DB=DB_DEVELOPMENT_DOCKER
     ports:
       - 5432:5432
 
   postgres_test:
     image: postgres:13.1
     environment:
-      - POSTGRES_USER=user
-      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=
```

Comment:
why did you change the env variables here?

Reply:
[#4 (comment)](https://github.com/Simba-City/nueno/pull/4#discussion_r843284630)